docs/building_the_project_with_dpcpp.rst

```diff
@@ -423,20 +422,20 @@ set, the backend libraries to enable the use of BLAS, LAPACK and RNG with MKLGPU
 and MKLCPU would also be enabled. The build of examples is disabled. Since
 functional testing was not disabled, tests would be built.
 
-Build oneMKL for the BLAS domain on a generic SYCL device:
+Build oneMath for the BLAS domain on a generic SYCL device:
 
 .. code-block:: bash
 
-  cmake $ONEMKL_DIR \ 
+  cmake $ONEMATH_DIR \ 
       -DCMAKE_CXX_COMPILER=clang++ \
       -DCMAKE_C_COMPILER=clang \ 
       -DENABLE_MKLCPU_BACKEND=False \ 
       -DENABLE_MKLGPU_BACKEND=False \
       -DENABLE_PORTBLAS_BACKEND=True
 
-Note that this is not a tested configuration. This builds oneMKL Interfaces
-with the portBLAS backend only, for a generic SYCL device supported by the 
-Open DPC++ project.
+Note that this is not a tested configuration. This builds oneMath with the
+portBLAS backend only, for a generic SYCL device supported by the Open DPC++
+project.
 
 Build oneMKL for the DFT domain on a generic SYCL device:
```

Comment:
This DFT example section still needs renaming in the same way as the BLAS example above (three mentions of "oneMKL")

Reply:
Thank you for catching them! This is fixed in https://github.com/oneapi-src/oneMKL/pull/602/commits/a42a97089dae9a1e672a1560b0bc4545cea64cd9